nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Updater/UpdaterWorker.cs

```diff
@@ -117,6 +122,20 @@ public async Task RunAsync(string repoRootPath, string workspacePath, string dep
         }
     }
 
+    private async Task RunForPackagesLockAsync(
+        string repoRootPath,
+        string lockPath)
+    {
+        _logger.Log($"Running for lock file [{Path.GetRelativePath(repoRootPath, lockPath)}]");
+        if (!File.Exists(lockPath))
+        {
+            _logger.Log($"File [{lockPath}] does not exist.");
+            return;
+        }
+
+        await Process.Start("dotnet", $"restore --force-evaluate {Path.GetDirectoryName(lockPath)}").WaitForExitAsync();
```

Comment:
Maybe use [`ProcessEx.RunAsync`](https://github.com/dependabot/dependabot-core/blob/de34b53e2e5cac84a4e0f2fd0bdb2276e01e1ca7/nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Utilities/ProcessExtensions.cs#L9) so that the process output is redirect to the same stream as the other times where `dotnet` is called externally?